frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetails.tsx

```diff
@@ -231,7 +230,7 @@ export function PolicyTemplateDetails() {
               id={'template-details-section'}
               title={kind + ' ' + t('details')}
               leftItems={descriptionItems}
-              defaultOpen={isCertPolicy}
+              defaultOpen
```

Comment:
Should this be set to `true` instead? I'm not seeing where this might be defined.

Reply:
I had to look this up too but this is magic syntax that is the equivalent of `defaultOpen={true}`.

Reply:
Just define  `defaultOpen`  == true 
Sorry for confusing you